adding-autodeployment/README.md

```diff
@@ -0,0 +1,100 @@
+# Autodeployment Guide
+## What to do?
+1. [Get deployer credentials](https://github.com/18F/development-guide/tree/add-autopilot/adding-autodeployment#1-getting-deployer-credentials)
+2. [Update `.travis.yml`](https://github.com/18F/development-guide/tree/add-autopilot/adding-autodeployment#2-update-travisyml)
+3. [Add `deploy.sh`](https://github.com/18F/development-guide/tree/add-autopilot/adding-autodeployment#3-add-deploysh)
+4. [Add manifests](https://github.com/18F/development-guide/tree/add-autopilot/adding-autodeployment#4-add-manifests)
+
+## What is this  guide for?
+This guide is to help developers implement [Contraband's Autopilot](https://github.com/contraband/autopilot) plugin on a [cloud.gov](https://cloud.gov/) hosted app. In the future we hope to provide you with a small script to generate this all for you!
+
+## Pre-reqs
+This guide assumes that you already have:
+- a git repository
+- continuous integration services on [Travis CI](https://travis-ci.org/).
+- a cloud.gov account.
+
+## 1. Getting deployer credentials
+Use the [instructions on Cloud.gov](https://cloud.gov/docs/apps/continuous-deployment/#provisioning-deployment-credentials) to create a deployer account for your app.
+
+In general, follow the rules of configuration as code. But because of `travis encrypt` escape issues, we suggest that you store these credentials in the travis environement variable gui found here: https://travis-ci.org/18F/YOUR_REPO/settings. Make sure that the *Display value in build log* setting is **OFF** for this value.
+
+## 2. Update `.travis.yml`
+If you are following from above, ignore the continuous integration instructions from above. We will be using a slightly different set of instructions from what is listed there.
+
+### before_deploy
+To utilize autopilot, you will need your travis deployment to have both cloud foundry and autopilot installed. To do this, you will modify your `before_deploy` section of your travis configuration file.
+
+First, look up on [cloud foundry's github](https://github.com/cloudfoundry/cli/releases) to find the latest release of the cf-cli for the **Linux 64 bit binary**.
```

Comment:
Good call on this step, for some reason it hadn't occurred to me.

I took a look at [the GitHub API](https://developer.github.com/v3/repos/releases/#get-the-latest-release), and it looks like https://api.github.com/repos/cloudfoundry/cli/tarball might always download the latest release. Not sure if that's reliable, though.